server/src/routes/api/v1/usersRouter.js

```diff
@@ -20,4 +21,16 @@ usersRouter.post("/", async (req, res) => {
   }
 });
 
+usersRouter.patch("/:id", uploadImage.single("image"), async (req, res) => {
+  try {
+    const user = req.user;
+
+    const body = await user.$query().patchAndFetch({ image: req.file.location });
+
+    return res.status(201).json({ body });
+  } catch (error) {
+    return res.status(500).json({ errors: error });
+  }
```

Comment:
I know its tedious, but you should handle for validation errors here, same as a POST request